packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -242,6 +242,35 @@ export class Pagination
     this.startItem = Math.max(1, this.startItem - this.pageSize);
   }
 
+  /**
+   * Set a specified page as active.
+   *
+   * @param page
+   */
+  @Method()
+  async goToPage(page: number): Promise<void> {
+    const totalPages = Math.ceil(this.totalPages);
+    if (page > totalPages) {
+      this.startItem = totalPages;
+    } else if (page < 1) {
+      this.startItem = 1;
+    } else {
+      this.startItem = page;
+    }
+  }
+
+  /** Set the first page as active. */
+  @Method()
+  async startPage(): Promise<void> {
```

Comment:
stepper.tsx has them as "goToStep", "startStep", and "endStep", but I'll change them if desired.

Reply:
Good discussion. We should definitely be consistent, but from an API-perspective, we should use verbs for methods and nouns for properties. I see the following options:

1. use the same name for consistency (following existing `stepper` ones)
2. consolidate these into a single method with the following signature:
    ```tsx
    goToPage(page: number | "start" |"end"): void;
    ```

We'd create a follow-up issue to revisit APIs for consistency regardless.

I'm kind of leaning towards 2 since it introduces a single method, it is named as a verb, it is concise, and is flexible.

Sidebar, I think we can close https://github.com/Esri/calcite-design-system/issues/6134. I don't see anything wrong with the convenience methods.

Reply:
Do we really even need `Page`? The component is `pagination`. Seems kind of repetitive to have `Page` in there.

`pagination.goTo("start")` seems sufficient, no?

or `pagination.page("start")`??? Since the act of using the component is paging?

Reply:
I do prefer a single method vs `end()`, `start()`, etc.

Reply:
@jcfranco what should I do regarding the following?
1. What @driskull commented above

Reply:
1. Good idea. We can drop `Page`.
2. What's the use case? Alternatively, we can clamp to the closest if the requested page is beyond the range.
3. That'd be a noop, methinks.

Reply:
For 2, it would be, for example, if there are only 10 pages but the caller requests page 12. Similarly, if somehow the user requests page 0.

Reply:
Maybe it should throw an error or log a warning and clamp the value to its limits?